client/src/pages/Home.js

```diff
@@ -14,20 +26,86 @@ export default function Home({ members, onOpenModal }) {
     }
   }
 
+  function filterGroups(event) {
+    const fieldValue = event.target.value;
+    setGroupValue(fieldValue);
+    const searchedGroup = availableGroups.filter(
+      (group) => group === fieldValue
+    );
+    setRenderedGroups(searchedGroup);
+  }
+
   return (
     <>
       <h2>Home</h2>
-      {orderedMembers.map((member) => (
-        <Member
-          key={member.id}
-          member={member}
-          onOpenModal={() => onOpenModal(member.id)}
-        />
+      <GroupFilter>
+        <label htmlFor="group">Filter group:</label>
+        <select id="group" value={groupValue} onChange={filterGroups}>
+          <option>Please select...</option>
+          {availableGroups.map((group) => (
+            <option>{group}</option>
+          ))}
+        </select>
+        <span onClick={() => setRenderedGroups(availableGroups)}>&times;</span>
+      </GroupFilter>
+      {renderedGroups.map((group) => (
+        <GroupWrapper>
+          <GroupHeadline>
+            {group}
+            <Delete onClick={() => deleteGroup(group)}>&times;</Delete>
+          </GroupHeadline>
+          {cantDeleteGroup === group && (
```

Comment:
ahh! now I get the naming 💡

Reply:
😃 it's probably not optimal - I will think about the naming again